app/keepers/keepers.go

```diff
@@ -299,14 +291,58 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		appKeepers.IBCKeeper.ChannelKeeper,
 	)
 
+	composablePrefix := sdk.GetConfig().GetBech32AccountAddrPrefix()
+
+	wasmDir := filepath.Join(homePath, "wasm")
```

Comment:
does this need to assume anything on how/where some artifact is stored? In that case, do we need to update any docs?

Reply:
I just moved line in file close to usage, it is not change. just refactoring to make wasmd setup in one place.